.circleci/config.yml

```diff
@@ -28,5 +28,50 @@ jobs:
             sudo mv protoc-gen-grpc-web /usr/local/bin/
             chmod +x /usr/local/bin/protoc-gen-grpc-web
 
-      - run: mvn clean verify jacoco:report
+      - run:
+          name: "Build And Run Unit Tests"
+          command: mvn clean install -DskipITs
+
+  ilp_integration_tests:
+    working_directory: ~/repo
+
+    # Need to run in a VM so TestContainers will work
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
Yes, it would.  One solution would be to just run the entire build in a VM, but it slows down the build, so I thought it would be better to build in a docker container and run just the ITs in a VM